### DISCUSSION TITLE
Does borg-export-tar take into account compression of the repo?

### DISCUSSION THREAD

**Discussion by aardbol** [borgbackup/borg#7785] [General]
I'd like to know whether it can take into account the compression method of the repo and just create an archive based on that setting, rather than to do double the effort. So if I want to create a zstd compressed tar archive with borg from a repo that's already compressed with zstd, for best efficiency I'd like it to create that archive without decompressing first and then compressing. Is that how it already works or is it possible to achieve?

**Comment by ThomasWaldmann** [comment]
That can't work like that, because:
- for borg, the chunks (content pieces of each file) are individually compressed
- for tar, the whole tar file (tar file metadata + whole file content concatenated for all files) is compressed

**Comment by aardbol** [comment]
Alright thanks, makes sense!
